app/src/main/java/com/nsc/covidscore/MainActivity.java

```diff
@@ -171,6 +224,20 @@ public void openNewRiskDetailPageFragment(CovidSnapshot cs, Location selectedLoc
         }
     }
 
+    public void rerunApis(Location location) {
```

Comment:
I think this code should have been removed in a previous PR? There should be a vm method you can call to update the database on lines 282, 291, 300 instead of calling rerunApis(), check with @dansjack since that was his PR

Reply:
Yep there's a view model method to rerun the api calls now. 

The method directly above rerunApis in this PR branch uses it  ![image](https://user-images.githubusercontent.com/30481844/99367502-e47f6580-286e-11eb-96ed-5ce077723377.png)

Reply:
Resolved